examples/mobile/android_basic/Cargo.toml

```diff
@@ -65,5 +65,5 @@ label = "Bevy Example"
 workspace = true
 
 [package.metadata.docs.rs]
-rustdoc-args = ["-Zunstable-options", "--cfg", "docsrs"]
+rustdoc-args = ["--cfg", "-Zunstable-options", "docsrs"]
```

Comment:
I scrolled a bit through and randomly spotted this, if we want to do something like this we need to be careful to exclude arrays where the order is meaningful.